### ISSUE TITLE
Change to Stable Branch

### ISSUE THREAD

**Issue by terrytibbles** [librenms/librenms#7520]
DO NOT DELETE THIS INFORMATION.

> Please read this information carefully.

GitHub issues is for known/validated bugs, please do not post issues asking for help or how to do X, Y or Z. 

You can use our irc channel ##librenms on freenode to ask questions or our [community site](https://community.librenms.org).

If you have a feature request please post this on our [community site](https://community.librenms.org/c/feature-requests).

Please confirm each of the sections below by putting an `x` in the box like `[x]`.

- [x] Is your install up to date? [Updating your install](http://docs.librenms.org/General/Updating/)
      Please do not submit an issue if your install is not up to date within the last 24 hours or on a stable monthly release.
- [x] Please include all of the information between the `====================================` section of `./validate.php` which you can run from the cli.
- [ ] Please provide ALL info asked for [here](http://docs.librenms.org/Support/FAQ/#faq20).
- [ ] Please provide as much detail as possible.
- [ ] Please do NOT post more than 10 lines of debug information here, use a pastebin service or GitHub Gists.

Love librenms? Please consider supporting our collective:
👉  https://t.libren.ms/donations

Hello,

I followed the instructions to install LibreNMS on Ubuntu (but installed it on Debian) and it seemed to work fine. If I go [here](https://docs.librenms.org/#General/Releases/) though and change to the stable branch, I get the following error:

```
root@debian:~# git fetch --tags && git checkout $(git describe --tags $(git rev-list --tags --max-count=1))
fatal: Not a git repository (or any of the parent directories): .git
root@debian:~#
```

```
====================================
Component | Version
--------- | -------
LibreNMS  | 1.32.01
DB Schema | 211
PHP       | 7.0.19-1
MySQL     | 10.1.26-MariaDB-0+deb9u1
RRDTool   | 1.6.0
SNMP      | NET-SNMP 5.7.3
====================================
```

Thanks.

**Comment by terrytibbles** [comment]
OK, I changed to the `/opt/librenms` directory and this seems to work fine! It might be worthwhile putting this in the documentation.

**Comment by laf** [comment]
This issue has been closed as it isn't classed as a bug or a new device request.

For help please use our IRC channel or the community site: http://docs.librenms.org/Support/FAQ/#faq3

**Comment by terrytibbles** [comment]
Fixed! #7525
